src/main/java/teammates/common/datatransfer/FeedbackQuestionAttributes.java

```diff
@@ -59,6 +59,27 @@ public FeedbackQuestionAttributes(FeedbackQuestion fq) {
         removeIrrelevantVisibilityOptions();
     }
 
+    public FeedbackQuestionAttributes(FeedbackQuestionAttributes other) {
```

Comment:
Just a thought, will it be better if we make a `getCopy` method and make the copy-constructor private? Something like:
```java
public FeedbackQuestionAttributes getCopy() {
    return FeedbackQuestionAttributes(this);
}

private FeedbackQuestionAttributes(FeedbackQuestionAttributes other) {
    // as per this method
}
```